frontend/src/components/deployments/display-code/DisplayCode.jsx

```diff
@@ -100,7 +100,7 @@ const DisplayCode = ({ isDialogOpen, setDialogOpen, url }) => {
     fetch("{{url}}", requestOptions)
     {{else}}
     var requestOptions = { method: 'GET', redirect: 'follow', headers: myHeaders};
-    fetch("{{url}}?execution_id=REPLACE_WITH_EXECUTION_ID", requestOptions)
+    fetch("{{url}}?execution_id=REPLACE_WITH_EXECUTION_ID&include_metadata=False", requestOptions)
```

Comment:
NIT:
@Deepak-Kesavan If we are using the same query string / params for all the calls here, we could might as well make this a constant.